### ISSUE TITLE
[Suggestion] Removing books from the tome

### ISSUE THREAD

**Issue by mathgeniuszach** [Vazkii/AkashicTome#44]
I don't mind the duplicated books feature already in-game (although it might be possible to check for matching nbt and meta in the items), or being able to add non-nbt books to the tome (it can be useful for roleplay or cosmetics), but it would be useful if I could remove entries from the book once already added. When I put too many items into the tome (or don't want a specific book in there anymore), I want to be able to remove an item from the list of books. Perhaps you could add this feature with a right click on a selected book in the GUI instead of a left click? I don't think this has been suggested before or is in the mod, correct me if I'm wrong.

**Comment by mathgeniuszach** [comment]
I suppose this suggestion would fit into the morph-o-tool mod as well.

**Comment by Tetrago** [comment]
Your question's a little hard to follow, but you can press Shift + Q to drop the select book from the tome, if that's what you're looking for.

**Comment by aziel13** [comment]
I think they are asking to be able to right click a book in the menu to remove it, without closing the menu.

**Comment by aziel13** [comment]
I think they are asking to be able to remove a book by right clicking it in the menu.
